pkg/utils/k8sutil.go

```diff
@@ -0,0 +1,74 @@
+package utils
```

Comment:
What's the difference between `helpers` package and `utils`?

Reply:
The name is different.

Reply:
Sorry to jump in, please confirm with reviewer that your answer has addressed the question before clicking "resolve", this apply to code review in general, my understanding of this question is: why we need `utils` since we already have `helpers`?

Reply:
Yeah, what we have been saying is everything under pkg/common are generically utility, helpers. K8s utils was introduced specifically for k8s functions.

Reply:
@kevdowney  is it OK we do this movement in a later PR since this PR mainly address using client reference?

Reply:
No, let’s not wait. I’m simply asking you to move k8sutil.go file under common dir

Reply:
a0f5e20382ee2907d097662dfd0d220dbdfb180a
Done